### DISCUSSION TITLE
Rerun tests when external package updates

### DISCUSSION THREAD

**Discussion by jeffsee55** [vitest-dev/vitest#2104] [Q&A]
👋 I'm working in a monorepo and I can't get Vitest to re-run when an adjacent package updates watchExclude doesn't seem to do what I want https://vitest.dev/config/#watchexclude. I tried just passing an empty array so it included everything but no luck
import { defineConfig } from 'vitest/config'

```
export default defineConfig({
  test: {
    // watchExclude: ['**/node_modules/**', '**/dist/**']
    watchExclude: [],
  },
})
```

**Comment by sheremet-va** [comment]
Is the file that changes directly imported in your test?

**Reply by jeffsee55** [reply]
@sheremet-va yes it is

**Reply by jeffsee55** [reply]
Not only does the test not re-run, but I have to kill the entire watch process and restart it.
